Remove style affecting images in email editor mobile preview [MAILPOET-6368]

lysyjan opened the pull request:
## Description

This PR removes styles that changed image size in the email preview for mobiles in the new email editor.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-6368]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes


[MAILPOET-6368]: https://mailpoet.atlassian.net/browse/MAILPOET-6368?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Preview

[Preview in WordPress Playground](https://account.mailpoet.com/playground/new/branch:email-editor/fix-image-mobile-preview)

_The latest successful build from `email-editor/fix-image-mobile-preview` will be used. If none is available, the link won't work._

costasovo reviewed (changes requested):
This fixes the issue with a small image, but reintroduces an issue when the fixed image size is set wider than mobile viewport. I think it should be fixable using max-width.
<img width="934" alt="Screenshot 2025-01-06 at 17 06 29" src="https://github.com/user-attachments/assets/38c9385d-662d-4ec4-9b9e-4e31cc9ac308" />

lysyjan committed:
Update styles for mobile preview

I removed some deprecated styles and updated the style affecting images in mobile preview.
[MAILPOET-6368]

lysyjan commented:
@costasovo Thank you for your review and help. I removed our styles affecting max-width and used max-width as you proposed.

costasovo reviewed (approved):
Thanks for the fix!

Aschepikov reviewed (approved):
GJ!
